src/plugins/validation/2and3/semantic-validators/operations-shared.js

```diff
@@ -119,6 +119,25 @@ module.exports.validate = function({ resolvedSpec, isOAS3 }, config) {
           });
         }
       }
+      const hasOperationTags = op.tags && op.tags.length > 0;
+      const hasGlobalTags = resolvedSpec.tags && resolvedSpec.tags.length > 0;
+      const resolvedTags = [];
+      if (hasOperationTags && hasGlobalTags) {
+        for (let i = 0; i < resolvedSpec.tags.length; i++) {
+          resolvedTags.push(resolvedSpec.tags[i].name);
+        }
+        for (let i = 0, len = op.tags.length; i < len; i++) {
+          if (!resolvedTags.includes(op.tags[i])) {
+            const checkStatus = config.unused_tag;
+            if (checkStatus !== 'off') {
+              result[checkStatus].push({
+                path: `paths.${pathKey}.${opKey}.tags`,
+                message: 'tag is not defined in operations'
```

Comment:
This is a case where I think it would be helpful to print the name of the unused tag in the message. Also, the tag is defined in operations but not the global level. So, the message is a little unclear. Something like this would be better:
```js
'tag is not defined at the global level: `op.tags[i]`'
```